### PR TITLE
Chore/add detailed update process

### PR DESCRIPTION
**Pull request by LuisRickert**
With this PR we improve the description of the "how to update a KIT" section by adding more information and detailed step description @maximilianong

### CODE REVIEW

**Review by FaGru3n — CHANGES_REQUESTED** [review]
Please remove any `SystemGuideOrg` related Pic's 
this is my personal TryOut Orga and should not be included into Eclipse Tractus X 😉

**Comment by LuisRickert** [comment]
Sure thing!

File: docs/kit-process/processes/update-documentation.md [hunk 1]
```diff
@@ -18,9 +18,32 @@ Currently the documentation has to be manually maintained by the process describ
...
+
+   4. Click on Create Pull request
+
+      ![IMG: Click on Create Pull Request second](resources/click-create-pr.png)
+
+   5. Add Description, what you add/change/improve. If available link an Issue with "#\<Issue Number\>. Then click on "Create Pull request"
+
+      ![IMG: Click on Create Pull Request final](resources/click-create-pr-final.png)
+
 4. The Traktus-X Team will review your pull request
```
Comment (FaGru3n) [inline docs/kit-process/processes/update-documentation.md:47]: little typo `Traktus-X`

**Review by FaGru3n — CHANGES_REQUESTED** [review]

**Comment by LuisRickert** [comment]
latest change: rebase onto main

**Comment by LuisRickert** [comment]
@FaGru3n please review

**Pull request merged**